File: 1012/1012.c
#include<stdio.h>

int main(){

    double a , b , c , tri , cir , trap , quad , ret , pi = 3.14159;
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);

    tri = (a * c)/2;
    cir = pi * (c * c);
    trap = ((a + b) * c)/ 2 ;
    quad = b * b;
    ret = a * b;

    printf("TRIANGULO: %.3lf\n" , tri);
    printf("CIRCULO: %.3lf\n", cir);
    printf("TRAPEZIO: %.3lf\n", trap);
    printf("QUADRADO: %.3lf\n", quad);
    printf("RETANGULO: %.3lf\n", ret);

    return 0;
}

File: URI/1012/1012.py
a , b , c = map(float , input().split())
pi = 3.14159

tri = (a * c)/2
cir = pi * (c * c)
trap = ((a + b) * c)/ 2 
quad = b**2
ret = a * b

print("TRIANGULO: %.3lf" % tri)
print("CIRCULO: %.3lf" % cir)
print("TRAPEZIO: %.3lf" % trap)
print("QUADRADO: %.3lf" % quad)
print("RETANGULO: %.3lf" % ret)